decidim/decidim
# Fix character counter with emoji picker close to maximum characters

#### :tophat: What? Why?
PR to make the emoji button disabled when there's not enough character left

#### :pushpin: Related Issues
- Fixes #8707 

#### Testing
- Write into a textarea until there are less than 4 characters left
- Notice that the emoji button is disabled
- Remove characters until there are more than 4 characters left
- Notice that the emoji button is enabled

#### :clipboard: Checklist



:hearts: Thank you!
